docs/proposals/multi-version.md

```diff
@@ -0,0 +1,53 @@
+# Multi-Version-Landscaper-Instances
+
+This document contains a proposal how to support multiple versions of Landscaper Instances in one Landscaper as a 
+Service (LaaS) landscape.
+
+## Requirements
+
+- A LaaS landscape should support different versions of a Landscaper instance (LI), i.e. different LandscaperDeployments 
+  install different versions of a [this](../../.landscaper/landscaper-instance) component. A more detailed
+  selection of particular deployer versions seems to be too complicated and out of scope.
+
+- A LaaS landscape supports a predefined set of LI versions, which change from release to realease (probably if the LaaS 
+  component version changes).
+
+- The Central Landscaper should always be on the latest Landscaper version. The version referenced by the 
+  LaaS project.  
+
+- A customer must be able 
+  - to select its intended LI version during onboarding. This must be optional. He gets the latest version
+    if nothing is selected.
+  - to upgrade to its intended LI version. Therefore, an allowed upgrade path must be enforced.
+  - see which versions are currently supported, which are recommended and which are outdated and what are the possible
+    upgrade paths.
+
+- How to upgrade a customer LI instance if his current version is not supported anymore? This might become even more 
+  critical if a security issue came up.
+  - Proposal for an initial solution: 
+    - Not supported versions are not upgraded. Only the reconciliation of the corresponding Installation is retriggered
```

Comment:
What does this mean exactly ? I would like to see that we never have unsupported LI versions running somewhere. Stakeholders must make sure that they upgrade to a supported LI version until their currently used LI version runs out of support. If they don't do this -> forced-upgrade ?

Reply:
lets discuss this later

Reply:
I have adapted this according to the result of our discussion: forced update for unsupported versions